docs/src/components/PageContent/PageContent.astro

```diff
@@ -38,4 +45,9 @@ import TableOfContents from '../RightSidebar/TableOfContents.tsx';
     <nav class="block sm:hidden">
         <MoreMenu editHref={githubEditUrl}/>
     </nav>
+    {
+      next && <aside>
+      Next Article: <a href={`${Astro.site.pathname}${next.link}`}>{next.text}</a>
```

Comment:
I18n might be needed? Not sure about this as there's no i18n function available as far as I can see.